docs/site/src/app/pages/docs-layout/docs-layout.component.ts

```diff
@@ -10,6 +10,8 @@ import {DocService} from "../../services/doc.service";
   styleUrls: ["./docs-layout.component.scss"]
 })
 export class DocsLayoutComponent implements OnInit {
+  innerWidth: number;
```

Comment:
We shouldn't use javascript to layout our page. Please replace this with its CSS equivalent. 

If it is unneeded please remove it instead.